js_modules/dagit/packages/core/src/assets/AssetPartitionDetail.tsx

```diff
@@ -0,0 +1,134 @@
+import {Box, Colors, Group, Heading, Icon, Mono, Subheading, Tag} from '@dagster-io/ui';
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+import {Timestamp} from '../app/time/Timestamp';
+import {isHiddenAssetGroupJob} from '../asset-graph/Utils';
+import {PipelineReference} from '../pipelines/PipelineReference';
+import {RunStatusWithStats} from '../runs/RunStatusDots';
+import {titleForRun, linkToRunEvent} from '../runs/RunUtils';
+import {isThisThingAJob, useRepository} from '../workspace/WorkspaceContext';
+import {buildRepoAddress} from '../workspace/buildRepoAddress';
+
+import {AssetEventMetadataEntriesTable} from './AssetEventMetadataEntriesTable';
+import {AllIndividualEventsLink} from './AssetEventsTable';
+import {AssetEventGroup} from './groupByPartition';
+
+export const AssetPartitionDetail: React.FC<{
+  group: AssetEventGroup;
+  hasLineage: boolean;
+}> = ({group, hasLineage}) => {
+  const {latest, partition, all} = group;
+  const run = latest?.runOrError?.__typename === 'Run' ? latest.runOrError : null;
+  const repositoryOrigin = run?.repositoryOrigin;
+  const repoAddress = repositoryOrigin
+    ? buildRepoAddress(repositoryOrigin.repositoryName, repositoryOrigin.repositoryLocationName)
+    : null;
+  const repo = useRepository(repoAddress);
+
+  const observationsAboutLatest =
+    latest?.__typename === 'MaterializationEvent'
+      ? group.all.filter(
+          (e) =>
+            e.__typename === 'ObservationEvent' && Number(e.timestamp) > Number(latest.timestamp),
+        )
+      : [];
+
+  return (
+    <Box padding={{horizontal: 24}} background={Colors.Gray10} style={{flex: 1}}>
+      <Box
+        padding={{vertical: 24}}
+        border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+        flex={{alignItems: 'center'}}
+      >
+        {partition ? (
+          <Box flex={{gap: 12}}>
+            <Heading>{partition}</Heading>
+            {latest ? <Tag intent="success">Materialized</Tag> : <Tag intent="none">Missing</Tag>}
+          </Box>
+        ) : (
+          <Heading>&nbsp;</Heading>
```

Comment:
An empty heading? Does this take up space? Is there a label we could give it instead?

Reply:
Ahh this was lazy of me, let me see what I can do. I wanted the empty state to be identical to the filled state, and this was a cheap way to keep the layout height the same.